src/Layout/PresentationMenu/PresentationSettings/PresentationSettings.tsx

```diff
@@ -123,23 +117,24 @@ export class PresentationSettingsMenu extends React.Component<PresentationSettin
         this.props.toggleMergeDistricts(event.target.checked);
 
         const year = this.props.year;
-        let election = this.props.electionType.elections.find((election) => election.year === year);
         let votes = this.props.votes.filter((vote) => vote.electionYear === year);
+        const tmpVotes = votes;
         const distributionYear = this.props.use2021Distribution && year >= 2005 ? 2021 : year;
         let metrics = this.props.metrics.filter((metric) => metric.electionYear === distributionYear);
         const parameters = this.props.parameters;
 
-        if (election !== undefined) {
+        if (votes.length > 0) {
             if (shouldDistributeDistrictSeats(year) && event.target.checked) {
-                election = mergeElectionDistricts(election, districtMap);
                 votes = mergeVoteDistricts(votes, districtMap);
                 metrics = mergeMetricDistricts(metrics, districtMap);
             }
+            if (tmpVotes !== votes) {
+                console.log("Changed");
```

Comment:
Codacy found an issue: [Unexpected console statement.](https://app.codacy.com/gh/Project-Lavinia/Lavinia-client/pullRequest?prid=5976811)

Reply:
Why do we have this?

Reply:
Accidentally left some debug statements.